rise-google-sheet.html

```diff
@@ -316,24 +314,11 @@
         if ( resp.request ) {
           /* eslint-disable indent */
           switch ( resp.request.status ) {
-            case 0:
```

Comment:
@rodrigopavezi I think we can do without the switch statement now and make it a if/else conditional block.